src/js/background/context-menu.js

```diff
@@ -374,6 +386,16 @@ const relayContextMenus = {
       );
       return relayAddresses;
     },
+    getPhoneMask: async () => {
+      // Note: This function only returns the mask saved in local storage.
+      //       That means that it will be out of date if this recently changed.
+      //       However, blocking the expansion of the context menu while waiting
+      //       for a GET request is not a great experience.
```

Comment:
praise: Agreed wholeheartedly. Since this mask number is _semi-permanent_ for the near future, this is very appropriate.